modules/synthetic-protocol/Cargo.toml

```diff
@@ -21,11 +21,11 @@ orml-utilities = { path = "../../orml/utilities", default-features = false }
 module-primitives = { path = "../primitives", default-features = false }
 traits = { package = "module-traits", path = "../traits", default-features = false }
 synthetic-tokens = { package = "module-synthetic-tokens", path = "../synthetic-tokens", default-features = false }
+orml-tokens = { path = "../../orml/tokens", default-features = false }
```

Comment:
tokens and currencies are dev-dependencies not dependencies

should be just this in dev-dependencies, also no need the std entries in features

```
orml-tokens = { path = "../../orml/tokens" }
orml-currencies = { path = "../../orml/currencies" }
```